### ISSUE TITLE
Unexpected error on save

### ISSUE THREAD

**Issue by shdobxr** [mschuchard/linter-ansible-linting#34]
I have set the ansible-lint executable path, but for some reason when saving a file I get the following error.

```
Traceback (most recent call last):
  File "/usr/local/bin/ansible-lint", line 6, in <module>
    from pkg_resources import load_entry_point
  File "/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py", line 3191, in <module>
    @_call_aside
  File "/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py", line 3175, in _call_aside
    f(*args, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py", line 3204, in _initialize_master_working_set
    working_set = WorkingSet._build_master()
  File "/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py", line 583, in _build_master
    ws.require(__requires__)
  File "/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py", line 900, in require
    needed = self.resolve(parse_requirements(requirements))
  File "/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py", line 786, in resolve
    raise DistributionNotFound(req, requirers)
pkg_resources.DistributionNotFound: The 'ansible-lint==4.1.0' distribution was not found and is required by the application
```
However I do have ansible-lint installed via brew.

Running atom 1.37.0 x64

Running ansible-lint manually does not throw any errors.

I have validated my path to ansible-lint, there are no other duplicates.

**Comment by shdobxr** [comment]
Solved me own problem. I had installed ansible-lint via brew. Which for some reason defaulted to install within py2.7. The fix was to remove it from brew `brew uninstall ansible-lint` then to install it with my pip3 utility thus placing the bits and bytes where they need to be.

Cheers. Closing issue.
